blazingcache-core/src/main/java/blazingcache/network/jvm/JVMChannel.java

```diff
@@ -44,13 +45,15 @@
 public class JVMChannel extends Channel {
 
     private static final Logger LOGGER = Logger.getLogger(JVMChannel.class.getName());
+    private static final AtomicLong ID_GENERATOR = new AtomicLong();
+    private static final AtomicLong CHANNEL_ID_GENERATOR = new AtomicLong();
     private volatile boolean active = false;
     private final Map<String, ReplyCallback> pendingReplyMessages = new ConcurrentHashMap<>();
     private final Map<String, Message> pendingReplyMessagesSource = new ConcurrentHashMap<>();
     private JVMChannel otherSide;
     private final ExecutorService callbackexecutor = Executors.newCachedThreadPool();
     private final ExecutorService executionserializer = Executors.newFixedThreadPool(1);
-    private String id = UUID.randomUUID().toString();
+    private String id = CHANNEL_ID_GENERATOR.incrementAndGet() + "";

```

Comment:
This ID is only used in the toString. In NettyChannel, the id was already a long

Reply:
if this is only a local variable, not related to the write protocol, you can change it to a "long"
btw generating JVMChannels shouldn't be on the hot paths (read/write), so it is not a big deal to change it